frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/DownloadDropdown.tsx

```diff
@@ -21,8 +21,8 @@ const DownloadDropdown: React.FC<DownloadDropdownProps> = ({
       position="right"
       toggle={
         <DropdownToggle
-          className="pf-v5-u-px-sm"
-          style={{ width: '60px' }}
+          // className="pf-v5-u-px-sm"
+          // style={{ width: '60px' }}
```

Comment:
cleanup comments

Reply:
oops, wasn't supposed to remove this in the first place. Thanks for pointing out.